Makefile

```diff
@@ -161,6 +161,14 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+.PHONY: yamllint
+yamllint:
+	yamllint -c .yamllint  deploy/examples/ --no-warnings
```

Comment:
```suggestion
	yamllint -c .yamllint deploy/examples/ --no-warnings
```

Reply:
Do you have whitespace visibility enabled in vim? I would sugget that to help avoid the spacing issues I keep seeing

Reply:
@BlaineEXE  wrote:

> Do you have whitespace visibility enabled in vim?

vim does not hide whitespaces best I can tell.
I would also need to research ways of visibly highlighting whitespaces. 



> I would suggest that to help avoid the spacing issues I keep seeing

I don't think that making whitespaces visible alone would prevent these issues from occurring.
I guess it  is more a point of me not seeing the changes I introduce unintentionally.
Some way of highlighting whitespace changes might help me.

Example: Many years ago, when working on a project that had banned trailing spaces from code files, I had configured vim to display them in bold red blocks. Maybe I can find something similar...